lib/fluent/plugin/in_systemd.rb

```diff
@@ -45,22 +67,22 @@ def shutdown
       private
 
       def init_journal
-        @journal = Systemd::Journal.new(path: @path)
+        @journal = ::Systemd::Journal.new(path: @path)
```

Comment:
We *shouldn't* need this change... did you run into some problems with this, or is this just being paranoid about getting the correct thing?